mesh/ncmesh.cpp

```diff
@@ -295,16 +296,30 @@ NCMesh::~NCMesh()
 #endif
 }
 
+void NCMesh::Node::SetScale(real_t s, bool overwrite)
```

Comment:
Probably worth checking that `0 < s < 1` so that you don't have coincident parent and child vertices, not sure what that would do to the mesh, but wouldn't be good.